docs/conf.py

```diff
@@ -112,38 +111,35 @@ def setup(app):
 }
 
 if os.environ.get("READTHEDOCS"):
-    # Get RTD build version ('latest' for master and actual version for tags)
-    # Use alias PKGVER to avoid mixin' with sphinx and wasting lots of time on debugging that
-    from packaging import version as PKGVER
-
-    rtd_ver = PKGVER.parse(os.environ.get("READTHEDOCS_VERSION"))
-
-    # see:
-    # https://docs.readthedocs.com/platform/stable/reference/environment-variables.html#envvar-READTHEDOCS_VERSION_TYPE
-    if os.environ.get("READTHEDOCS_VERSION_TYPE") == "external":
-        # Build and install NEURON from source
-        subprocess.run(
-            "cd .. && python setup.py build_ext bdist_wheel",
-            shell=True,
-            check=True,
-        )
-        subprocess.run(
-            f"pip install {glob.glob('../dist/*.whl')[0]}",
-            shell=True,
-            check=True,
-        )
-    else:
-        # Install neuron accordingly (nightly for master, otherwise incoming version)
-        # Note that neuron wheel must be published a priori.
-        subprocess.run(
-            "pip install neuron{}".format(
-                f"=={rtd_ver.base_version}"
-                if isinstance(rtd_ver, PKGVER.Version)
-                else "-nightly"
-            ),
-            shell=True,
-            check=True,
-        )
-
-    # Execute & convert notebooks + doxygen
-    subprocess.run("cd .. && python setup.py docs", check=True, shell=True)
+
+    # Execute & convert notebooks + doxygen (RTD calls sphinx on its own)
```

Comment:
Previous setup was too convoluted (sphinx is called via Python, then Sphinx calls Python _again_ to build the docs, but that Python actually just calls CMake, which calls Python _yet again_ to build the Python HOC module), so I removed a layer of Python.